lib/beaker-answers/helpers.rb

```diff
@@ -1,12 +1,12 @@
-def flatten_keys_to_joined_string(h, key_delim='::')
+def flatten_keys_to_joined_string(h, key_delim = '::')
   flat = {}
   h.each_pair do |k, v|
     if v.respond_to?(:keys)
       flatten_keys_to_joined_string(v, key_delim).each_pair do |k2, v2|
-        flat.merge!({[k.to_s, k2.to_s].join(key_delim) => v2})
+        flat.merge!([k.to_s, k2.to_s].join(key_delim) => v2)
```

Comment:
Drop this one.

Reply:
https://rubocop.readthedocs.io/en/latest/cops_style/#stylebracesaroundhashparameters disabled

Reply:
Or maybe for consistency, we should enforce that there are braces?